sdc/src/sdc/sdc.d

```diff
@@ -75,14 +75,29 @@ final class SDC {
 		semantic.terminate();
 		backend.visit(semantic.buildMain(modules));
 	}
-	
-	void codeGen(string objFile) {
+
+	void runLLVMPasses() {
```

Comment:
I don't think this should be exposed at all. It needs to run when generation any kind of file, so functions that generate other files, so these function should run the passes on a per needed basis.